src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 // write code here
+document.addEventListener('click', (e) => {
+  const clickedElement = e.target;
+  const list = clickedElement.querySelector('ul');
+
+  if (list.style.display !== 'none') {
```

Comment:
Accessing `style.display` directly might not always give the expected result if the display property is not set inline. Consider using `getComputedStyle` to get the actual computed style of the element.